src/Scouting.elm

```diff
@@ -1,141 +1,198 @@
-            }
-
-        DriverStationInput input ->
-            { model
-                | driverStation = input
-            }
-
-        Start ->
-            { model
-                | isStarted = not model.isStarted
-            }
-
-
-view : Model -> Html.Html Msg
-view model =
-    Html.div []
-        [ Html.text "Pre-Scout screen:"
-        , Html.div []
-            [ Html.input
-                [ placeholder "Scouter's name..."
-                , onInput NameInput
-                , value model.scouterName
-                ]
-                []
-            , Html.input
-                [ placeholder "Team's number..."
-                , onInput TeamInput
-                , value <|
-                    if 0 == model.teamNum then
-                        ""
-
-                    else
-                        String.fromInt model.teamNum
-                ]
-                []
-            , Html.input
-                [ placeholder "Match number..."
-                , onInput MatchInput
-                , value <|
-                    if 0 == model.matchNum then
-                        ""
-
-                    else
-                        String.fromInt model.matchNum
-                ]
-                []
-            , Html.input
-                [ style "visibility" dropDown_driverStation
-                , placeholder "Driver station..."
-                , onInput DriverStationInput
-                , value model.driverStation
-                ]
-                []
-            , Html.button [ onClick Start ]
-                [ Html.text <|
-                    if model.isStarted then
-                        "Started"
-
-                    else
-                        "Startn't"
-                ]
-            ]
-        , Html.div []
-            [ Html.text <|
-                "Status:  name - "
-                    ++ model.scouterName
-                    ++ ", team - "
-                    ++ String.fromInt model.teamNum
-                    ++ ", match - "
-                    ++ String.fromInt model.matchNum
-                    ++ ", station - "
-                    ++ model.driverStation
-            ]
-        ]
+module Scouting exposing (Model, Msg, driverStations, init, main, update, view)
+
+import Browser
+import Html
+import Html.Attributes exposing (placeholder, style, value)
+import Html.Events exposing (onClick, onInput)
+import List.Extra exposing (elemIndex, getAt)
+import Maybe exposing (withDefault)
+import Maybe.Extra exposing (unwrap)
+import Tuple exposing (first, second)
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view
+        , update = update
+        }
+
+
+type Msg
+    = NameInput String
+    | TeamInput String
+    | MatchInput String
+    | Start
+
+
+type alias Model =
+    { scouter : String
+    , team : Maybe Int
+    , match : Maybe Int
+    , driverStation : String

```

Comment:
See https://github.com/orbit1690/Scouting-2019/pull/4#discussion_r353353898

Reply:
I think that the scouter's name can be whatever- they really enjoy it-, and the driver station is updated from a list of the 6 possible options.

Reply:
The scouter's name can be whatever - that's not an issue.
However, the driver station is an issue. Right now it seems "safe", but code changes frequently, so the invariant (the 6 valid options) should be enforced at type-level.